scipy/stats/_morestats.py

```diff
@@ -3957,29 +3958,19 @@ def directionalmean(samples, *, axis=0, nan_policy='propagate'):
     >>> directionalmean(data)
     array([1., 0., 0.])
 
-    The `regular` sample mean in contrast does not lie on the unit sphere.
+    The *regular* sample mean in contrast does not lie on the unit sphere.
 
     >>> data.mean(axis=0)
     array([0.8660254, 0., 0.])
 
     """
     samples = np.asarray(samples)
-    if nan_policy not in ['raise', 'propagate', 'omit']:
-        raise ValueError("nan_policty must be one of 'propagate', 'omit', "
-                         "'raise'.")
     if samples.ndim < 2:
         raise ValueError("samples must at least be two-dimensional. "
                          f"Instead samples has shape: {samples.shape!r}")
+    if samples.shape[-1] < 2:
```

Comment:
I don't think 1D is a good use of this function, but it still makes sense and works correctly, right? Personally, I wouldn't add this limitation if it already does the right thing because it's just more work for us.

Reply:
For 1D input I expect the function to return `array([1.])` as it would just divide the mean by its magnitude. I think this would only confuse users. In an even worse scenario, a user might expect that a circular mean is computed and also does not get the desired output.

Reply:
Now that I imagine the real line with a vector pointing in + or - direction, this totally makes sense. Removed that check.